NSS-Day-Cohort-42/rare-media-moguls
# User can now edit post from details page

#### Changes Made
1. Modified file `PostForm.js` to include edit mode through useEffect
1. Modified `ApplicationViews.js` to include path routed to `PostForm` view from and edited view
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout sm-edit-post
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
>Navigate to a single post page to view post detail like http://localhost:3000/posts/4
>Click edit post button to check that it navigates you to a new edit page.
>That page should populate a form with the post details already entered
>Make a change to the post and then click save
>You should be redirected to that post's detail page and you should see the updated information
